dts/msm8916/msm8916-samsung-r11.dts

```diff
@@ -20,6 +20,14 @@
 		compatible = "samsung,gprimeltezt", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-bootloader = "G530Y*";
 
+		panel {
```

Comment:
Do you really need it for a single entry here?

Reply:
Some of the Grand Prime e.g. SM-G530FZ/H/W variants have 2 panel entries but this one has only one.